tensorrt_llm/_torch/modules/kimi_k3_moe/kimi_k3_moe_gate.py

```
@@ -0,0 +1,278 @@
+            )
+        return DeepSeekV3MoeRoutingMethod(
+            top_k=self.top_k,
+            n_group=self.num_expert_group,
+            topk_group=self.topk_group,
+            routed_scaling_factor=self.routed_scaling_factor,
+            callable_e_score_correction_bias=lambda: self.e_score_correction_bias,
+            is_fused=True,
+        )
+
+    def forward(self, hidden_states: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
+        bsz, seq_len, h = hidden_states.shape
```

Ruff (0.16.1)

[warning] Unpacked variable `h` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)